azure/iac/aks.rego

```diff
@@ -665,3 +665,55 @@ aks_kub_dashboard_disabled_metadata := {
     "Policy Help URL": "",
     "Resource Help URL": "https://docs.microsoft.com/en-us/azure/templates/microsoft.containerservice/managedclusters"
 }
+
+
+#
+# PR-AZR-ARM-AKS-010
+#
+
+default aks_local_account_disabled = null
+#Defaults to false
+azure_attribute_absence["aks_local_account_disabled"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.containerservice/managedclusters"
+    not resource.properties.disableLocalAccounts
+}
+
+azure_issue["aks_local_account_disabled"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.containerservice/managedclusters"
+    resource.properties.disableLocalAccounts != true
+}
+
+aks_local_account_disabled {
+    lower(input.resources[_].type) == "microsoft.containerservice/managedclusters"
+    not azure_attribute_absence["aks_local_account_disabled"]
+    not azure_issue["aks_local_account_disabled"]
+}
+
+aks_local_account_disabled = false {
+    azure_attribute_absence["aks_local_account_disabled"]
+}
+
+aks_local_account_disabled = false {
+    azure_issue["aks_local_account_disabled"]
+}
+
+aks_local_account_disabled_err = "microsoft.containerservice/managedclusters property 'disableLocalAccounts' is missing from the resource" {
+    azure_attribute_absence["aks_local_account_disabled"]
+} else = "Azure Kubernetes Service Clusters currently dont have local authentication methods disabled" {
+    azure_issue["aks_local_account_disabled"]
+}
```

Comment:
we should change the error massages like this:
```
aks_local_account_disabled_err = "Azure Kubernetes Service Clusters currently dont have local authentication methods disabled" {
    azure_issue["aks_local_account_disabled"]
} else = "microsoft.containerservice/managedclusters property 'disableLocalAccounts' is missing from the resource" {
    azure_attribute_absence["aks_local_account_disabled"]
}
```
priority of ```azure_issue``` more than ```azure_attribute_absence```
please update all other similar rules like this.

Reply:
@shahinshirvani i don't think that matters. as far as i know there is no priority. both will not fail at the same time.
any other reason that i dont know?

Reply:
@shahinshirvani got your point, its happening when we are not using `has_property `custom function.